site/en/tutorials/images/classification.ipynb

```diff
@@ -960,7 +960,7 @@
       },
       "outputs": [],
       "source": [
-        "image_gen = ImageDataGenerator(rescale=1./255, zoom_range=0.5)"
+        "image_gen = ImageDataGenerator(rescale=1./255, zoom_range=0.5) # zoom_range from 0 - 1 where 1 = 100"
```

Comment:
Since this is narrative doc, can you add this to the text section above to make this clearer?

Reply:
Yes sure